src/crl.rs

```diff
@@ -484,64 +486,76 @@ impl<'a> BorrowedRevokedCert<'a> {
 
 impl<'a> FromDer<'a> for BorrowedRevokedCert<'a> {
     fn from_der(reader: &mut untrusted::Reader<'a>) -> Result<Self, Error> {
-        der::nested(reader, Tag::Sequence, Error::BadDer, |der| {
-            // RFC 5280 §4.1.2.2:
-            //    Certificate users MUST be able to handle serialNumber values up to 20 octets.
-            //    Conforming CAs MUST NOT use serialNumber values longer than 20 octets.
-            //
-            //    Note: Non-conforming CAs may issue certificates with serial numbers
-            //    that are negative or zero.  Certificate users SHOULD be prepared to
-            //    gracefully handle such certificates.
-            // Like the handling in cert.rs we choose to be lenient here, not enforcing the length
-            // of a CRL revoked certificate's serial number is less than 20 octets in encoded form.
-            let serial_number = lenient_certificate_serial_number(der)
-                .map_err(|_| Error::InvalidSerialNumber)?
-                .as_slice_less_safe();
-
-            let revocation_date = Time::from_der(der)?;
-
-            let mut revoked_cert = BorrowedRevokedCert {
-                serial_number,
-                revocation_date,
-                reason_code: None,
-                invalidity_date: None,
-            };
-
-            // RFC 5280 §5.3:
-            //   Support for the CRL entry extensions defined in this specification is
-            //   optional for conforming CRL issuers and applications.  However, CRL
-            //   issuers SHOULD include reason codes (Section 5.3.1) and invalidity
-            //   dates (Section 5.3.2) whenever this information is available.
-            if der.at_end() {
-                return Ok(revoked_cert);
-            }
+        der::nested(
+            reader,
+            Tag::Sequence,
+            Error::TrailingData("BorrowedRevokedCert::outer"),
```

Comment:
Maybe instead of `outer`, `crlEntry`?

Reply:
Replaced it with `BorrowedRevokedCert::crl_entry`.